ccd-definition/CaseTypeTab/5_CasePeople.json

```diff
@@ -58,5 +58,15 @@
     "TabDisplayOrder": 5,
     "CaseFieldID": "representatives",
     "TabFieldDisplayOrder": 6
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "Channel": "CaseWorker",
+    "TabID": "CasePeopleTab",
+    "TabLabel": "Allocated Judge",
+    "TabDisplayOrder": 5,
```

Comment:
Not sure if it's in the AC but could double check with BA display order incase they want judge higher up in the tab

Reply:
Will do :)

Reply:
he said put at top :)